### PR TITLE
laurel-about.7.md minimal typos

### PR DESCRIPTION
**Pull request by rurban**
Nice talk at BSides Dresden!

### CODE REVIEW

File: man/laurel-about.7.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ In addition to (or instead of) writing log files, `auditd(8)` can pass log lines
...
+Log records carrying the same event ID (the `msg=audit(TIME:SEQUENCE):` part) are collected into coherent events and output as a JSON lines-based log format. Most importantly, hex-encoded strings are output as regular JSON strings. [RfC8259] mandates that "text exchanged between systems that are not part of a closed ecosystem MUST be encoded using UTF-8", therefore any bytes or byte sequences that are not valid UTF-8 are percent-encoded as described in [RfC3986]. Numbers are parsed as decimal, octal, or hexadecimal values and output in an unambiguous format. List data (`SYSCALL.{a0 … a3}` and `EXECVE.a*`) are turned into JSON arrays. `PROCTITLE.proctitle` is split at NULL bytes and transformed into a list.
```
Comment (hillu) [inline man/laurel-about.7.md:48]: Thanks. It should probably be "JSON Lines"

Comment (rurban) [reply man/laurel-about.7.md:48]: Na. lines-based is lowercase

Comment (hillu) [reply man/laurel-about.7.md:48]: I also should put in a reference to https://jsonlines.org/ – Thanks for the hints, I'll close the PR.

**Comment by rurban** [comment]
I see, you were right

**Pull request closed without merging**